common/xrdp_constants.h

```diff
@@ -562,13 +566,13 @@
 #define XR_CODEC_GUID_H264 \
   "\x48\x32\x36\x34\x00\x00\x10\x00\x80\x00\x00\xAA\x00\x38\x9B\x71"
 
-#define RDP_CAPSET_SURFCMDS       0x1c
-#define RDP_CAPLEN_SURFCMDS       0x0c
-#define RDP_CAPSET_BMPCODECS      0x1d
-#define RDP_CAPLEN_BMPCODECS      0x1c
+#define RDP_CAPSET_SURFCMDS       0x1C
+#define RDP_CAPLEN_SURFCMDS       0x0C
+#define RDP_CAPSET_BMPCODECS      0x1D
+#define RDP_CAPLEN_BMPCODECS      0x1C
 #define RDP_CAPSET_COMPDESK       0x19
 #define RDP_CAPLEN_COMPDESK       0x06
-#define RDP_CAPSET_LPOINTER       0x27
+#define RDP_CAPSET_LPOINTER       0x1B
```

Comment:
It this change intentional? Other changes in this file are for formatting only, this one changes the value.